pkg/controller/config/config_controller.go

```diff
@@ -722,3 +731,20 @@ func sourceBasedOnRecursion(path string) mf.Source {
 	}
 	return mf.Path(path)
 }
+
+func createAddonsPipelines(mgr manager.Manager) (mf.Manifest, error) {
+	addonsPath := filepath.Join("pkg/utils/addons", "pipelinetemplate.yaml")
+	pipelineTemplate, err := mf.NewManifest(addonsPath)
+	if err != nil {
+		return mf.Manifest{}, err
+	}
+	pipelines, err := addons.CreatePipelines(pipelineTemplate)
+	if err != nil {
+		return mf.Manifest{}, err
+	}
+	m1, err := mf.ManifestFrom(mf.Slice(pipelines),mf.UseClient(mfc.NewClient(mgr.GetClient())))
+	if err != nil {
+		return mf.Manifest{}, err
```

Comment:
could you move this like to the above function, and make `addons.CreatePipeliens` return `manifest`